stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -704,12 +742,12 @@ public struct ${Self}<Pointee>
   /// bound to the `Pointee` type.
   ///
   /// - Parameter count: The number of instances to deinitialize. `count` must
-  ///   not be negative. The default value is `1`.
+  ///   not be negative. 
   /// - Returns: A raw pointer to the same address as this pointer. The memory
   ///   referenced by the returned raw pointer is still bound to `Pointee`.
   @_inlineable
   @discardableResult
-  public func deinitialize(count: Int = 1) -> UnsafeMutableRawPointer {
+  public func deinitialize(count: Int) -> UnsafeMutableRawPointer {
```

Comment:
This can’t be overloaded because deinitialize(count:) with the default arg resolves to the same symbol as deinitialize(count:) without the default arg.

Reply:
You can add a no-argument method that forwards to the new version, and make it obsoleted in 5.

Reply:
isn’t that a completely new symbol?

Reply:
Yes, but that’s fine.

Reply:
done